### DISCUSSION TITLE
Why requests to "miktex.org" domain does not work in codespace?

### DISCUSSION THREAD

**Discussion by breno-jesus-fernandes** [community/community#57663] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Can't install MikTeX because requests to miktex.org domain are not working, anyone can help me to undestand if it is possible to unblock this domain in codespace?

**Comment by chriskthomas** [comment]
Try a mirror: https://mirror.ctan.org/systems/win32/miktex/

**Reply by chriskthomas** [reply]
E.g. https://mirror.mwt.me/ctan/systems/win32/miktex/setup/deb/pool/universe/m/miktex/miktex_23.5-jammy1_amd64.deb

**Reply by breno-jesus-fernandes** [reply]
That's strange ... I could not install the .deb, a lot of problems with dependecies ... Did you have luck?

**Reply by chriskthomas** [reply, marked as answer]
oops. You need: https://mirror.las.iastate.edu/tex-archive/systems/win32/miktex/setup/deb/pool/universe/m/miktex/miktex_23.5-focal1_amd64.deb

Try
```bash
wget "https://mirror.las.iastate.edu/tex-archive/systems/win32/miktex/setup/deb/pool/universe/m/miktex/miktex_23.5-focal1_amd64.deb"
sudo apt update && sudo apt install ./miktex_23.5-focal1_amd64.deb
```

**Reply by chriskthomas** [reply]
that is, you need focal instead of jammy

**Reply by breno-jesus-fernandes** [reply]
I tried with bullseye, but not with focal, that's it. thanks.

**Reply by chriskthomas** [reply]
Could you mark it as answered so that people with the same issue find it more easily?
